root/static/styles/forms.less

```
@@ -751,3 +751,75 @@ div.rel-editor-dialog {
 button.guess-timezone {
     background-image: data-uri('../images/icons/magnet.png');
 }
+
+.dialog-root {
+    .modal-backdrop {
+        background: #000;
+        position: fixed;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        opacity: 0.1;
+    }
+}
+
+.dialog {
+    visibility: hidden;
+    padding: 1em;
+    position: absolute;
+    background: @text-white;
+    border: 1px solid #AAA;
+    top: 0;
+    left: 0;
+    filter: drop-shadow(2px 2px 4px @very-dark-grey);
+    -webkit-filter: drop-shadow(2px 2px 4px @very-dark-grey);
+
+    h1 {
+        color: @musicbrainz-orange;
+        margin-bottom: 1em;
+    }
+
+    &.modal {
+        max-width: 500px;
+
+        @media @wide {
+            max-width: 700px;
+        }
+    }
+
+    &.popover {
+        max-width: 500px;
```

Maybe round corners the same as with AC bubble:
```suggestion
    &.popover {
        max-width: 500px;
        .border-radius(12px);
```